serverlessworkflow/antora.yml

```diff
@@ -21,16 +21,18 @@ asciidoc:
     #quarkus_platform: com.redhat.quarkus.platform
     quarkus_platform: io.quarkus.platform
     # upstream: kogito-quarkus-serverless-workflow
-    # kogito_sw_gav: org.kie.kogito:kogito-quarkus-serverless-workflow:1.24.0.Final-redhat-00001
-    kogito_sw_gav: kogito-quarkus-serverless-workflow
-    # downstream: 2.7.6.Final-redhat-00006
+    # kogito_sw_ga: org.kie.kogito:kogito-quarkus-serverless-workflow
+    kogito_sw_ga: kogito-quarkus-serverless-workflow
+    # downstream: 2.7.6.Final-redhat-00011
```

Comment:
non-existent rhbq version, see https://mvnrepository.com/artifact/com.redhat.quarkus.platform/quarkus-bom

Reply:
Hmm, this is right actually. Good catch!